components/inputs/input-time.js

```diff
@@ -168,12 +168,16 @@ class InputTime extends LitElement {
 						aria-controls="${this._dropdownId}"
 						aria-labelledby="${this._dropdownId}-label"
 						@change="${this._handleChange}"
+						@keydown="${this._handleKeydown}"
 						class="d2l-input"
 						?disabled="${this.disabled}"
-						@keypress="${this._handleKeypress}"
```

Comment:
Am I missing something... not seeing `_handleKeypress` anywhere in the code...

Reply:
Neither am I, and I don't remember using it for anything, which confuses me 🤔 maybe it ended up in there initially for some debug purpose and nobody noticed in previous PRs.

Reply:
I'm guessing since `this._handleKeyPress` is `undefined`, Lit is smart enough to just not wire up the handler and not throw.